opentera_webrtc_demos/launch/opentera_demo.launch

```
@@ -105,6 +125,16 @@
     <!-- Robot status simulated data -->
     <node name="robot_status" type="robot_status.py" pkg="opentera_webrtc_demos" />
 
-    <node pkg="opentera_webrtc_robot_gui" type="opentera_webrtc_robot_gui_node" name="robot_gui" output="$(arg output)" respawn="true"/>
+    <!-- Loopback or webcam camera -->
+    <node if="$(arg use_outgoing_face_cropping)" pkg="cv_camera" type="cv_camera_node" name="camera1">
+        <param name="device_id" type="int" value="$(arg camera_id)"/>
+        <param name="rate" type="double" value="30"/>
+        <param name="cv_cap_prop_fps" type="double" value="30"/>
```

This is unreliable: setting the framerate to a value that is not supported by the webcam will fail silently.

---

I just removed them, it matches other uses of cv_camera and it still works